x/common/dec.go

```diff
@@ -10,7 +10,7 @@ import (
 
```

Comment:
Made the code more descriptive here and added tests for it. Since the `Sqrt` is used in `x/perp`, it's important to make sure these functions don't panic.